README.md
# college-book-history-analysis
作用：分析大学期间读过的书

使用 `parse.py` 提取出个人图书馆读书记录页面的所有读书记录
![借书记录片段](read_history_files/read-history-example.png)
```javascript
{
    "name": "JavaScript和jQuery实战手册",
    "number": "1",
    "operation": "还书",
    "code": "03398387"
  },
  {
    "name": "软件驱魔 : 调试和优化遗留代码的艺术",
    "number": "2",
    "operation": "还书",
    "code": "03476473"
  },
  {
    "name": "CSS实战手册 : 第2版 = =CSS:The Missing Manual, Second Edition",
    "number": "3",
    "operation": "还书",
    "code": "03269496"
  },
  {
    "name": "CSS实战手册 : 第2版 = =CSS:The Missing Manual, Second Edition",
    "number": "4",
    "operation": "借书",
    "code": "03269496"
  },
  {
    "name": "JavaScript和jQuery实战手册",
    "number": "5",
    "operation": "借书",
    "code": "03398387"
  }
  
  ...
```

使用 `getTags.js` 获取读书记录的豆瓣标签

```javascript
{count: 38, name: 'web开发', title: 'web开发'},
  {count: 35, name: '前端', title: '前端'},
  {count: 30, name: 'Web前端', title: 'Web前端'},
  {count: 24, name: '前端开发', title: '前端开发'},
  {count: 20, name: 'web', title: 'web'},
  {count: 12, name: 'css', title: 'css'},
  {count: 7, name: '计算机', title: '计算机'},
  {count: 9, name: 'JavaScript', title: 'JavaScript'},
  {count: 6, name: '前端', title: '前端'},
  {count: 4, name: 'web', title: 'web'},
  {count: 4, name: 'jquery', title: 'jquery'},
  {count: 3, name: '计算机', title: '计算机'},
  {count: 3, name: '前端开发', title: '前端开发'},
  {count: 2, name: 'kindle', title: 'kindle'},
  {count: 2, name: 'Web', title: 'Web'},
  {count: 1248, name: '社会学', title: '社会学'},
  {count: 671, name: '贾雷德·戴蒙德', title: '贾雷德·戴蒙德'},
  {count: 438, name: '环境', title: '环境'},
  {count: 419, name: '人类学', title: '人类学'},
  {count: 416, name: '社会', title: '社会'},
  {count: 341, name: '历史', title: '历史'},
  {count: 236, name: '环境保护', title: '环境保护'},
  {count: 218, name: '"崩溃"', title: '"崩溃"'},
  {count: 1947, name: '林语堂', title: '林语堂'},
  {count: 853, name: '生活的艺术', title: '生活的艺术'},
  {count: 707, name: '随笔', title: '随笔'},
  {count: 641, name: '散文', title: '散文'},
  
  ...
```

使用 `buildWordCloud.py` 根据豆瓣标签生成 *词云* 
![my-book-history-in-collage-word-cloud.jpg](analysis-result/my-book-history-in-collage-word-cloud.jpg)

history.txt
图书题名
CSS实战手册 : 第2版
JavaScript和jQuery实战手册
软件调试实战
崩溃 : 社会如何选择成败兴亡
生活的艺术
领域驱动设计 : 软件核心复杂性应对之道
程序员必读之软件架构
软件驱魔 : 调试和优化遗留代码的艺术
简单之美：软件开发实践者的思考
程序员度量 : 改善软件团队的分析学
程序员应该知道的97件事
微服务设计
沙发上的精神分析师
论人的本性
当代华语电影人物论
Google工作整理术 : 信息太多、时间太少、Google帮你做整理
结构性思维
学习、创造与使用知识 : 概念图促进企业和学校的学习变革
卓有成效的程序员
实战Gradle
皮格马利翁效应
微服务设计
持续交付 : 发布可靠软件的系统方法
软件调试实战
持续集成 : 软件质量改进和风险降低之道
HTTP权威指南
沙发上的精神分析师
结构性思维
学习、创造与使用知识 : 概念图促进企业和学校的学习变革
当代华语电影人物论
论人的本性
Google工作整理术 : 信息太多、时间太少、Google帮你做整理
卓有成效的程序员
实战Gradle
微服务设计
皮格马利翁效应
HTTP权威指南
系统化思维导论
程序开发心理学 : 银年纪念版
中国的品格
Computer concepts :
皮格马利翁效应
HTTP权威指南
Computer concepts :
物联网科技导论
MS Office 办公软件整 合应用档问候语教程
摄影构图圣经
数码摄影构图实战
考研单词一笑而过
JSP程序设计 : 慕课版
C程序设计实例详解
农业产业化龙头企业全面风险管理体系研究 : 以吉林省为例
网络营销那些事儿 : 顶级营销专家的实战案例分享
睿炼 : 互联网+时代的商业逻辑
皮格马利翁效应
HTTP权威指南
Node.js实战 . 第2季[专著]
JavaScript Web应用开发
七周七Web开发框架
程序员的职业素养
本书书名无法描述本书内容 : 一场逗你玩的哲学探险
Java RESTful Web Service 实战
测试驱动开发的艺术
中国人大历史
Computer concepts :
第一本Docker书
本书书名无法描述本书内容 : 一场逗你玩的哲学探险
测试驱动开发的艺术
Java RESTful Web Service 实战
测试驱动的JavaScript开发
第一本Docker书
Computer concepts :
中国人大历史
Java 8函数式编程
全端Web开发 : 使用JavaScript与Java
JavaScript应用程序设计 : 使用Node、HTML5和现代JavaScript类库打造稳健的web应用
七周七语言 : 理解多种编程范型
Docker进阶与实战
重新认识你自己
每个人都会死，但我总以为自己不会
重构与模式
算法设计与分析基础
测试驱动开发的艺术
卓有成效的程序员
实现模式
程序设计实践
写给大忙人看的Java SE 8
测试驱动的JavaScript开发
Java RESTful Web Service 实战
函数式编程思维
计算的本质 : 深入剖析程序和计算机
重构与模式
人生中不可不想的事
测试驱动开发的艺术
算法设计与分析基础
卓有成效的程序员
实现模式
程序员的职业素养
程序设计实践
情绪的语言
编程珠玑
程序开发心理学 : 银年纪念版
Effective JavaScript : 编写高质量JavaScript代码的68个有效方法
React : 引领未来的用户界面开发框架
程序员的思维修炼开发认知潜能的九堂课
算法
Linux命令行大全
函数式编程思维
七周七Web开发框架
编程珠玑
Node.js实战
JUnit实战
领域特定语言
全端Web开发 : 使用JavaScript与Java
测试驱动开发 : 实战与模式解析
Java程序员修炼之道
写给大忙人看的Java SE 8
计算机网络自顶向下方法与Internet特色
算法设计与分析基础
算法
写给大家看的设计书 : 实例与创意
学会解决问题 : 支持问题解决的学习环境设计手册
情绪的语言
测试驱动开发的艺术
编程珠玑
Compiler construction priciples and practice /Kenneth C. Louden.
Java程序员修炼之道
写给大忙人看的Java SE 8
超越Java:探讨程序语言的未来
怎样解题
Agile Java中文版
测试驱动开发 : 实战与模式解析
测试驱动开发的艺术
算法
编译器设计
算法设计与分析基础
C程序设计的抽象思维
JUnit实战
计算机系统要素:从零考试构建现代计算机
程序设计实践
实现模式
卓有成效的程序员
编程珠玑
Compiler construction priciples and practice /Kenneth C. Louden.
怎样解题
编码 : 隐匿在计算机软硬件背后的语言
Agile Java中文版
编译器设计
算法引论——一种创造性方法
算法
算法设计与分析基础
C语言的科学和艺术
C程序设计的抽象思维
Java程序设计—一种跨学科的方法
程序设计实践
实现模式
卓有成效的程序员
UNIX编程艺术
TCP/IP详解 , 卷1:协议
编程珠玑
深入浅出Networking
编译器设计
算法设计与分析基础
算法引论——一种创造性方法
算法
Agile Java中文版
怎样解题
编码 : 隐匿在计算机软硬件背后的语言
嵌入式系统软硬件协同设计实战指南 : 基于Xilinx Zynq
Xilinx All Programmable Zynq-7000 SoC设计指南
自制编程语言
敏捷软件开发 : 原则、模式与实践
程序员修炼之道 : 从小工到专家
逻辑的引擎
Agile Java中文版
编码 : 隐匿在计算机软硬件背后的语言
怎样解题
算法设计与分析基础
算法
算法引论——一种创造性方法
Xilinx All Programmable Zynq-7000 SoC设计指南
嵌入式系统软硬件协同设计实战指南 : 基于Xilinx Zynq
图解TCP/IP
Linux命令行大全
计算机系统概论
计算机系统
计算机组成:结构化方法
C interfaces and implementations :
算法设计与分析基础
算法引论——一种创造性方法
程序设计实践
算法
编码 : 隐匿在计算机软硬件背后的语言
Orange’S : 一个操作系统的实现
程序员的思维修炼开发认知潜能的九堂课
怎样解题
当尼采哭泣
计算机专业英语
Agile Java中文版
计算机系统概论
计算机组成:结构化方法
计算机系统
编码 : 隐匿在计算机软硬件背后的语言
Orange’S : 一个操作系统的实现
JAVA并发编程实践
程序员的思维修炼开发认知潜能的九堂课
怎样解题
当尼采哭泣
Effective Java 中文版
皮格马利翁效应
Linux命令行大全
汇编语言
Agile Java中文版
Ruby基础教程
Ruby从入门到精通
Java程序员修炼之道
超越Java:探讨程序语言的未来
皮格马利翁效应
Effective Java 中文版
Linux命令行大全
情绪的语言
Agile Java中文版
深入浅出Ajax : 中文版
深入浅出JavScript
JSP高级程序设计
图解TCP/IP
汇编语言
习惯的力量 : 为什么我们这样生活，那样工作
三体
自控力
Orange’S : 一个操作系统的实现
21天学通Linux C编程
UML精粹:标准对象建模语言简明指南
Agile Java中文版
深入浅出JavScript
JSP高级程序设计
深入浅出Ajax : 中文版
程序员修炼之道 : 从小工到专家
思考的乐趣 : Matrix67数学笔记

Core Java :
程序员的思维修炼开发认知潜能的九堂课
编程珠玑
计算机专业英语
Java语言程序设计
怎样解题
重构与模式
新概念计算机英语
自控力
把时间当作朋友
把时间当作朋友
Java语言程序设计
计算机专业英语
新概念计算机英语
Core Java :

怎样解题
批判的思考
批判性思维工具
自控力
编程珠玑
重构与模式
程序员的思维修炼开发认知潜能的九堂课
听透英语其实很重要
Java程序设计经典300例
批判性思维工具
批判的思考
习惯的力量 : 为什么我们这样生活，那样工作
摇摆 : 难以抗拒的非理性诱惑 : 新版
本能Ⅱ : 为什么我们总跟自己过不去？
心理经济学 : 心态决定人生的幸福度和国家的经济状况
Java程序设计经典300例
疯狂Java讲义精粹
听透英语其实很重要
How to win friends & influence people :
改变心理学的40项研究
程序设计语言原理
21天学通Linux C编程
程序员的思维修炼开发认知潜能的九堂课
大话数据库
Linux C函数库详解词典
六个思考框
改变你一生的心理技巧 : 年轻人必须懂得的沟通、交流和心理技巧
数学之美
正则表达式入门经典
算法框架理论 : 一门统一的社会科学之基础
发现数据之美 : 数据分析原理与实践
Android开发与应用
大嘴巴漫谈数据挖掘
英语流行话题阅读 : 语境识词3500
程序员的思维修炼开发认知潜能的九堂课
大话数据库
情绪的语言
数学建模方法与分析
Eclipse编程技术与实例
Eclipse 核心编程:插件、界面与Web开发
Eclipse插件开发
Eclipse插件开发学习笔记
大嘴巴漫谈数据挖掘
发现数据之美 : 数据分析原理与实践
英语流行话题阅读 : 语境识词3500
Android开发与应用
Android 应用程序设计
Android应用程序设计教程
什么是数学
更多的人死于心碎:我最喜爱的悲情小说
世界名人论辩技巧欣赏
呻吟语
图灵的秘密
Effective C++中文版:改善程序与设计的55个具体做法
Eclipse 核心编程:插件、界面与Web开发
Eclipse插件开发学习笔记
Eclipse插件开发
Eclipse编程技术与实例
C++面向对象程序设计
C++面向对象程序设计
面向对象程序设计综合实践
Java程序设计应用教程
什么是数学
启发每个人思维的数学小书
怪诞心理学 : 不可思议的心理操控.2 : 新版
皮格马利翁效应
改变心理学的40项研究
怪诞心理学 : 不可思议的心理操控.2 : 新版
皮格马利翁效应
改变心理学的40项研究
大一微积分
数理统计导论
数理统计导论
大一微积分
改变心理学的40项研究
大人都不知道的男女科学
怪诞习惯透露什么
复杂性中的思维:物质、精神和人类的复杂动力学
皮格马利翁效应
张爱玲文集:精读本
古文观止译注.上
张爱玲文集:精读本
皮格马利翁效应
复杂性中的思维:物质、精神和人类的复杂动力学
古文观止译注.上
改变心理学的40项研究
我这一代东京人
开国元勋:李光耀传
Photoshop CS3中文版完全剖析 : 彩印
改变心理学的40项研究
品三国 . 下
张爱玲文集:精读本
朱光潜美学文集.3
伊豆的舞女
谁？
世界名人论辩技巧欣赏
改变心理学的40项研究
怪诞心理学 : 不可思议的心理操控.2 : 新版
皮格马利翁效应
Robinson Crusoe
数学沉思录
用脑记还是用纸记 : 你的记忆能力超乎你的想象
改变心理学的40项研究
C语言课程设计案例精编
怪诞心理学 : 不可思议的心理操控.2 : 新版
皮格马利翁效应
关键词:文化与社会的词汇
C语言课程设计案例精编
皮格马利翁效应
C算法.第一卷,基础、数据结构、排序和搜索
抽象代数讲义
挑战智力水平的100道趣题
中国现代文化泰斗大讲坛.1,蔡元培讲教育
设计中的色彩心理学
设计中的色彩心理学
C++ Primer Plus中文版
C++ Primer Plus中文版
十一种孤独
虚实之间
麦田里的守望者
十一种孤独
麦田里的守望者
虚实之间
C++程序设计
C++程序设计
十一种孤独
认识心理学
围城
十一种孤独
围城
认识心理学
品三国 . 上
21天学通C++：第6版
C++ Primer Plus中文版
C和指针
痛并快乐着
钱钟书选集,散文卷
痛并快乐着
钱钟书选集,散文卷
C++ Primer Plus中文版
C和指针
21天学通C++：第6版
大学何为
大学学习论:大学生学习与成才指导
什么是数学
天生就会跑
野火集:二十年纪念版
虚实之间
天使与魔鬼
论衡.上
30天写小说
国史大纲 : 修订本 . 上册
国史大纲 : 修订本 . 下册
富兰克林自传
高等数学解题与分析
1000个思维游戏.上
高等数学解题与分析
C语言程序设计(Visual C++6.0环境)
C语言实例解析精粹
C语言程序设计(Visual C++6.0环境)
史记.下
高等数学习题详解:上、下册合订本
什么是数学
C语言程序设计(Visual C++6.0环境)
论人类不平等的起源和基础
怎样读文学古籍
高等数学 . 上册
高等数学 . 上册 , 理工类
高等数学解题与分析
陈寅恪家世
中华骄子,辛亥烈士
野火集:二十年纪念版
高等数学同步精讲.上册
策略性思维:个人和专业发展的有力工具
科学的历程:第二版
西方科技十二讲
NBA体能训练 : 专家设计的122项练习
思维版图
论人类不平等的起源和基础
策略性思维:个人和专业发展的有力工具


tags_1.js
 [{ count: 119, name: 'CSS', title: 'CSS' },
  { count: 38, name: 'web开发', title: 'web开发' },
  { count: 35, name: '前端', title: '前端' },
  { count: 30, name: 'Web前端', title: 'Web前端' },
  { count: 24, name: '前端开发', title: '前端开发' },
  { count: 20, name: 'web', title: 'web' },
  { count: 12, name: 'css', title: 'css' },
  { count: 7, name: '计算机', title: '计算机' } ,
 { count: 9, name: 'JavaScript', title: 'JavaScript' },
  { count: 6, name: '前端', title: '前端' },
  { count: 4, name: 'web', title: 'web' },
  { count: 4, name: 'jquery', title: 'jquery' },
  { count: 3, name: '计算机', title: '计算机' },
  { count: 3, name: '前端开发', title: '前端开发' },
  { count: 2, name: 'kindle', title: 'kindle' },
  { count: 2, name: 'Web', title: 'Web' } ,
 { count: 1248, name: '社会学', title: '社会学' },
  { count: 671, name: '贾雷德·戴蒙德', title: '贾雷德·戴蒙德' },
  { count: 438, name: '环境', title: '环境' },
  { count: 419, name: '人类学', title: '人类学' },
  { count: 416, name: '社会', title: '社会' },
  { count: 341, name: '历史', title: '历史' },
  { count: 236, name: '环境保护', title: '环境保护' },
  { count: 218, name: '"崩溃"', title: '"崩溃"' } ,
 { count: 1947, name: '林语堂', title: '林语堂' },
  { count: 853, name: '生活的艺术', title: '生活的艺术' },
  { count: 707, name: '随笔', title: '随笔' },
  { count: 641, name: '散文', title: '散文' },
  { count: 509, name: '哲学', title: '哲学' },
  { count: 437, name: '生活', title: '生活' },
  { count: 363, name: '中国文学', title: '中国文学' },
  { count: 292, name: '修养', title: '修养' } ,
 { count: 251, name: '领域驱动', title: '领域驱动' },
  { count: 161, name: '软件设计', title: '软件设计' },
  { count: 116, name: '软件工程', title: '软件工程' },
  { count: 93, name: 'DDD', title: 'DDD' },
  { count: 92, name: '架构', title: '架构' },
  { count: 70, name: '设计模式', title: '设计模式' },
  { count: 59, name: '软件开发', title: '软件开发' },
  { count: 48, name: '计算机', title: '计算机' } ,
 { count: 190, name: '软件架构', title: '软件架构' },
  { count: 110, name: '架构', title: '架构' },
  { count: 61, name: '软件开发', title: '软件开发' },
  { count: 55, name: '软件工程', title: '软件工程' },
  { count: 43, name: '编程', title: '编程' },
  { count: 28, name: '计算机', title: '计算机' },
  { count: 25, name: '计算机科学', title: '计算机科学' },
  { count: 21, name: '技术', title: '技术' } ,
 { count: 16, name: '调试', title: '调试' },
  { count: 13, name: '重构', title: '重构' },
  { count: 11, name: '软件工程', title: '软件工程' },
  { count: 8, name: '质量管理', title: '质量管理' },
  { count: 6, name: '编程', title: '编程' },
  { count: 4, name: '编程艺术', title: '编程艺术' },
  { count: 3, name: '程序员', title: '程序员' },
  { count: 2, name: '软件', title: '软件' } ,
 { count: 49, name: '软件工程', title: '软件工程' },
  { count: 27, name: '软件开发', title: '软件开发' },
  { count: 22, name: '软件开发实践者的思考', title: '软件开发实践者的思考' },
  { count: 19, name: '简单之美', title: '简单之美' },
  { count: 11, name: '问题&思考&思维', title: '问题&思考&思维' },
  { count: 11, name: '架构', title: '架构' },
  { count: 10, name: 'IT', title: 'IT' },
  { count: 9, name: '深度思考', title: '深度思考' } ,
 { count: 31, name: '软件工程', title: '软件工程' },
  { count: 15, name: 'O\'Reilly', title: 'O\'Reilly' },
  { count: 11, name: '软件开发', title: '软件开发' },
  { count: 11, name: '管理', title: '管理' },
  { count: 6, name: '项目管理', title: '项目管理' },
  { count: 6, name: '计算机', title: '计算机' },
  { count: 5, name: '度量', title: '度量' },
  { count: 5, name: '2013', title: '2013' } ,
 { count: 106, name: '程序设计', title: '程序设计' },
  { count: 79, name: '计算机', title: '计算机' },
  { count: 71, name: '软件开发', title: '软件开发' },
  { count: 67, name: '编程', title: '编程' },
  { count: 49, name: '程序员', title: '程序员' },
  { count: 41, name: '程序员应该知道的97件事', title: '程序员应该知道的97件事' },
  { count: 29, name: '软件工程', title: '软件工程' },
  { count: 21, name: 'O\'Reilly', title: 'O\'Reilly' } ,
 { count: 301, name: '微服务', title: '微服务' },
  { count: 232, name: '架构', title: '架构' },
  { count: 150, name: '软件架构', title: '软件架构' },
  { count: 84, name: '计算机', title: '计算机' },
  { count: 72, name: '软件工程', title: '软件工程' },
  { count: 56, name: '编程', title: '编程' },
  { count: 55, name: '技术', title: '技术' },
  { count: 53, name: '互联网', title: '互联网' } ,
 { count: 4, name: '软件工程', title: '软件工程' } ,
 { count: 50, name: '调试', title: '调试' },
  { count: 23, name: '计算机', title: '计算机' },
  { count: 22, name: '编程', title: '编程' },
  { count: 14, name: 'debug', title: 'debug' },
  { count: 10, name: '程序设计', title: '程序设计' },
  { count: 8, name: '软件调试实战', title: '软件调试实战' },
  { count: 8, name: '计算机科学', title: '计算机科学' },
  { count: 5, name: '技术', title: '技术' } ,
 { count: 143, name: '持续集成', title: '持续集成' },
  { count: 104, name: '软件工程', title: '软件工程' },
  { count: 61, name: '敏捷开发', title: '敏捷开发' },
  { count: 33, name: '项目管理', title: '项目管理' },
  { count: 28, name: '软件开发', title: '软件开发' },
  { count: 20, name: '计算机', title: '计算机' },
  { count: 20, name: 'CI', title: 'CI' },
  { count: 18, name: 'SCM', title: 'SCM' } ,
 { count: 7, name: '精神分析', title: '精神分析' },
  { count: 5, name: '心理学', title: '心理学' },
  { count: 2, name: '法国', title: '法国' },
  { count: 2, name: '心理', title: '心理' },
  { count: 1, name: '精神分析及其研究', title: '精神分析及其研究' },
  { count: 1, name: '社哲文化', title: '社哲文化' },
  { count: 1, name: 'W', title: 'W' } ,
 { count: 43, name: '思维', title: '思维' },
  { count: 31, name: '思维方法', title: '思维方法' },
  { count: 23, name: '结构性思维', title: '结构性思维' },
  { count: 13, name: '实用；思路清晰；好书，值得一读', title: '实用；思路清晰；好书，值得一读' },
  { count: 10, name: '自我管理', title: '自我管理' },
  { count: 6, name: '管理', title: '管理' },
  { count: 6, name: '好书，值得一读', title: '好书，值得一读' },
  { count: 6, name: '哲学', title: '哲学' } ,
 { count: 23, name: '思维', title: '思维' },
  { count: 22, name: '方法论', title: '方法论' },
  { count: 21, name: '学习', title: '学习' },
  { count: 17, name: '概念图', title: '概念图' },
  { count: 14, name: '学习方法', title: '学习方法' },
  { count: 8, name: '教育', title: '教育' },
  { count: 6, name: 'Visualization', title: 'Visualization' },
  { count: 3, name: '心理学认知心理学', title: '心理学认知心理学' } ,
 { count: 2, name: '电影', title: '电影' },
  { count: 1, name: '演员', title: '演员' },
  { count: 1, name: '导演', title: '导演' } ,
 { count: 20, name: '社会学', title: '社会学' },
  { count: 18, name: '生物学', title: '生物学' },
  { count: 12, name: '科普', title: '科普' },
  { count: 11, name: '爱德华·威尔逊', title: '爱德华·威尔逊' },
  { count: 7, name: '科学', title: '科学' },
  { count: 6, name: '进化论', title: '进化论' },
  { count: 6, name: '进化', title: '进化' },
  { count: 5, name: '人性', title: '人性' } ,
 { count: 97, name: '个人管理', title: '个人管理' },
  { count: 82, name: '工作管理', title: '工作管理' },
  { count: 63, name: 'Google', title: 'Google' },
  { count: 56, name: '自我管理', title: '自我管理' },
  { count: 54, name: '时间管理', title: '时间管理' },
  { count: 35, name: '工作', title: '工作' },
  { count: 30, name: '方法', title: '方法' },
  { count: 24, name: '工具', title: '工具' } ,
 { count: 419, name: '程序员', title: '程序员' },
  { count: 415, name: '软件开发', title: '软件开发' },
  { count: 385, name: '效率', title: '效率' },
  { count: 325, name: '编程', title: '编程' },
  { count: 182, name: '思维', title: '思维' },
  { count: 174, name: '计算机', title: '计算机' },
  { count: 128, name: 'programming', title: 'programming' },
  { count: 120, name: '程序设计', title: '程序设计' } ,
 { count: 21, name: 'gradle', title: 'gradle' },
  { count: 18, name: '自动化构建', title: '自动化构建' },
  { count: 14, name: 'Java', title: 'Java' },
  { count: 8, name: '软件开发', title: '软件开发' },
  { count: 5, name: '软件工程', title: '软件工程' },
  { count: 5, name: 'groovy', title: 'groovy' },
  { count: 4, name: '软件', title: '软件' },
  { count: 3, name: '项目构建', title: '项目构建' } ,
 { count: 115, name: '心理学', title: '心理学' },
  { count: 36, name: '心理', title: '心理' },
  { count: 32, name: '精神分析', title: '精神分析' },
  { count: 25, name: '人际关系', title: '人际关系' },
  { count: 16, name: '自我实现的预言', title: '自我实现的预言' },
  { count: 16, name: '社会心理', title: '社会心理' },
  { count: 14, name: '认识自己', title: '认识自己' },
  { count: 11, name: '皮格马利翁效应', title: '皮格马利翁效应' } ,
 { count: 1928, name: 'HTTP', title: 'HTTP' },
  { count: 898, name: '网络', title: '网络' },
  { count: 658, name: '计算机', title: '计算机' },
  { count: 568, name: '互联网', title: '互联网' },
  { count: 528, name: '协议', title: '协议' },
  { count: 463, name: 'web开发', title: 'web开发' },
  { count: 266, name: '基础理论', title: '基础理论' },
  { count: 205, name: '编程', title: '编程' } ,
 { count: 685, name: '思维', title: '思维' },
  { count: 462, name: '系统论', title: '系统论' },
  { count: 398, name: '系统思维', title: '系统思维' },
  { count: 222, name: '温伯格', title: '温伯格' },
  { count: 121, name: '软件工程', title: '软件工程' },
  { count: 109, name: '系统科学', title: '系统科学' },
  { count: 108, name: '计算机', title: '计算机' },
  { count: 105, name: '系统', title: '系统' } ,
 { count: 14, name: '计算机', title: '计算机' },
  { count: 10, name: '项目管理', title: '项目管理' },
  { count: 9, name: '编程', title: '编程' },
  { count: 9, name: '温伯格', title: '温伯格' },
  { count: 8, name: '软件工程', title: '软件工程' },
  { count: 8, name: '认知科学', title: '认知科学' },
  { count: 7, name: '软件开发', title: '软件开发' },
  { count: 7, name: '管理', title: '管理' } ,
 { count: 49, name: '中国文化', title: '中国文化' },
  { count: 39, name: '楼宇烈', title: '楼宇烈' },
  { count: 34, name: '文化', title: '文化' },
  { count: 23, name: '国学', title: '国学' },
  { count: 21, name: '哲学', title: '哲学' },
  { count: 20, name: '中国的品格', title: '中国的品格' },
  { count: 17, name: '中国', title: '中国' },
  { count: 12, name: '思想', title: '思想' } ,
 { count: 118, name: 'programming', title: 'programming' },
  { count: 104, name: '计算机科学', title: '计算机科学' },
  { count: 56, name: '编程', title: '编程' },
  { count: 48, name: '计算机', title: '计算机' },
  { count: 39, name: '程序设计', title: '程序设计' },
  { count: 35, name: '经典', title: '经典' },
  { count: 21, name: 'cs', title: 'cs' },
  { count: 20, name: 'Programming', title: 'Programming' } ,
 { count: 153, name: '物联网', title: '物联网' },
  { count: 37, name: 'IT', title: 'IT' },
  { count: 33, name: '网络', title: '网络' },
  { count: 30, name: '计算机', title: '计算机' },
  { count: 24, name: '科普', title: '科普' },
  { count: 24, name: '教材', title: '教材' },
  { count: 20, name: '商业', title: '商业' },
  { count: 18, name: '技术', title: '技术' } ]
